core/src/main/scala/org/apache/spark/h2o/utils/H2OContextUtils.scala

```diff
@@ -51,6 +54,64 @@ private[spark] trait H2OContextUtils extends Logging {
     }
   }
 
+  private def isTcpPortAvailable(port: Int): Boolean = {
+    val serverSocket = new ServerSocket()
+    try {
+      serverSocket.setReuseAddress(false)
+      val host = SparkEnv.get.blockManager.blockManagerId.host
+      val socketAddress = new InetSocketAddress(InetAddress.getByName(host), port)
+      serverSocket.bind(socketAddress, 1)
+      true
+    } catch {
+      case _: Exception => false
+    } finally {
+      try {
+        serverSocket.close()
+      } catch {
+        case _: Exception =>
```

Comment:
nit: `scala.util.Try` or `scala.util.control.Exception.ignoring`?

Reply:
Thanks for the hint, I have used `scala.util.Try` & like it!